README.md

```diff
@@ -1 +1,196 @@
 # Event-triggered Notifications Tutorial
+
+An event-triggered notification is an automated messages sent in response to an event.
+
+There are three approaches for implementing these types of notifications:
+
+- Leverage **Segments** for sending messages.
```

Comment:
Maybe I'm missing something, but this doesn't seem like three distinct techniques, this seems like: 
"Automating the Grouping of Users into Segments", "Tagging through the ... API", "Tagging in the ... SDK".  

In other words, segments seems like they stand out in this set, which suggests that this isn't a MECE list.  Is it a list of steps with options?  Give it some thought.